app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html

```diff
@@ -429,4 +433,7 @@ <h2>Copy</h2>
 {% block script %}
     {{ block.super }}
     <script src="{% static "js/refresh_sidebar.js" %}"></script>
+    {{ workstation_domains|json_script:"workstation-domains" }}
+    <script type="module" src="{% static 'workstations/js/session-control.mjs' %}"></script>
+
```

Comment:
Could make this one include from a partial template, may make updating easier.

Reply:
You mean putting these two lines in a partial template and then including that template everywhere where we are currently using these two lines, and not just here, right?  Good idea.